zipline/data/us_equity_pricing.py

```diff
@@ -767,35 +767,38 @@ def __init__(self, calendar, panel):
             # Fake volume if it does not exist.
             panel.loc[:, :, 'volume'] = int(1e9)
 
-        self.first_trading_day = panel.major_axis[0]
-        self._calendar = calendar
+        self.first_trading_day = normalize_date(panel.major_axis[0])
```

Comment:
`PanelBarReader` isn't constructed with a trading calendar; it's given `calendar`, but that's a list of execution datetimes.